src/lib/components/BlogPage.svelte

```diff
@@ -0,0 +1,198 @@
+<script lang="ts">
+  import Icon from "@iconify/svelte";
+  import { Select } from "bits-ui";
+  import { createEventDispatcher } from "svelte";
+  import { slide } from "svelte/transition";
+  import Tag from "./Tag.svelte";
+  import BlogLink from "$lib/components/BlogLink.svelte";
+  import { getEntries } from "$lib/utils/consts";
+  import { tags } from "$lib/utils/types/blogs";
+  import type { EndPoints } from "$lib/utils/types/microcms";
+
+  export let more: boolean = false;
+  export let blogs: EndPoints["gets"]["blogs"];
+  export let limit: number = 10;
+
+  let blogListContents = blogs.contents;
+
+  let sortKey: boolean = false;
+  let filterTag: (typeof tags)[keyof typeof tags]["title"] = "All";
+
+  console.log(blogs.contents);
+
+  let page = 1;
+
+  $: start = (page - 1) * limit;
+  $: end = start + limit;
+
+  const dispatch = createEventDispatcher();
+
+  function handlePublishedAtClick() {
```

Comment:
いつ使うかを表す `handlePublishedAtClick` より、処理の内容を説明する `toggleSortOrder` などの方がいいかも

```suggestion
  function toggleSortOrder() {
```
※これはやや宗教なので必ずしも従う必要はない